docs/reference/synonyms/apis/synonyms-apis.asciidoc

```diff
@@ -0,0 +1,31 @@
+[[synonym-management-apis]]
+== Synonym Management APIs
+
+beta::[]
+
+++++
+<titleabbrev>Synonyms Management APIs</titleabbrev>
+++++
+
+---
+
+Synonyms Management APIs let you create and manage synonyms sets.
+
+Synonyms sets can be used to configure <<analysis-synonym-graph-tokenfilter,synonym graph token filters>> and <<analysis-synonym-tokenfilter,synonym token filters>>.
+
+Once a synonyms set is used in a synonym token filter, these token filters can be used as part of a <<search-analyzer,search analyzers>>.
+You can dynamically change the synonyms used at search time using these APIs to update the synonyms sets.
+Your search results will immediately reflect the synonyms set change.
+
+The following APIs can be used to manage synonyms sets:
+
+* <<put-synoyms-set>>
```

Comment:
<<put-synoyms-set>> => <<put-synonyms-set>> and others

Reply:
thanks!